concent-vm/helper-scripts/concent-update.sh

```diff
@@ -0,0 +1,24 @@
+#! /bin/bash -e
+
+DEFAULT_BRANCH=master
```

Comment:
If would be best if the default branch was the same here and in the playbook that sets up concent.

Now one defaults to `master` and the other to the version from `versions.yml`.